From 88e74ee4dc0e1d16fb1c476f59607b9df5ea1cce Mon Sep 17 00:00:00 2001
From: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>
Date: Mon, 7 Jul 2025 12:32:25 +0000
Subject: [PATCH] CIV: sensor hal wo static list

Enable sensor mediation hal
without static sensor list
reporting.

Tracked-On: OAM-92946
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>
Signed-off-by: Kishan Mochi <kishan.mochi@intel.com>
Signed-off-by: Basanagouda Nagappa Koppad <Basanagouda.Nagappa.Koppad@intel.com>

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 37d017d1..955d12fb 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -84,7 +84,7 @@ atrace: false
 firmware: true(all_firmwares=false)
 aaf: true
 suspend: never
-sensors: mediation(enable_sensor_list=true)
+sensors: mediation(disable_static_sensor_list=true)
 bugreport: true
 mainline-mod: true
 houdini: true
-- 
2.34.1

